### PR TITLE
[FUT1-17705] Ccr0232 s02 model changes

### PR DESCRIPTION
**Pull request by Oana-Vasile**
https://support.systematic.com/browse/FUT1-17705
* ModelChanges-1: Change cardinality from 0...1 to 0...* on ehealth-actionguidance and ehealth-view. | S2
* ModelChanges-2: Add useContext extension with cardinality 0..* to DocumentReference and CareTeam (reuse useContext extension from ehealth-actionguidance and ehealth-view

### CODE REVIEW

File: input/intro-notes/StructureDefinition-ehealth-careteam-intro.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ Instead CareTeam is intended to be referenced from CarePlan and EpisodeOfCare. A
  a single CarePlan but the typical scenario is that it is
 referenced by multiple CarePlan and/or EpisodeOfCare instances crossing multiple Patient instances.
 
+The profile is based on the native FHIR Basic resource and adds the following extensions:
```
Comment (sys-alex-matic) [inline input/intro-notes/StructureDefinition-ehealth-careteam-intro.md:20]: I suggest to do something similar to activityDefinition where it contains:
image

File: input/pagecontent/changelog.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ This is the log of changes made to the eHealth Implementation Guide.
 ### General changes
 - Changed cardinality for ehealth-servicerequest performer to 0..*
 - Added ehealth-careteam reference to ehealth-servicerequest performer.
+- Changed cardinality for ehealth-usecontext and ehealth-view to 0..*
```
Comment (sys-alex-matic) [inline input/pagecontent/changelog.md:8]: Changed useContext cardinality for ehealth-actionguidance and ehealth-view to 0..*

File: input/intro-notes/StructureDefinition-ehealth-careteam-intro.md (hunk 1 above)
Comment (Oana-Vasile) [reply input/intro-notes/StructureDefinition-ehealth-careteam-intro.md]: I am not sure if I understand completely.
Would this be ok:

The eHealth profile of CareTeam has the following extensions:
* `ehealth-useContext` which defines the context that the content is intended to support

Or would you like more text in the ehealth-useContext description?

Comment (esn-systematic) [reply input/intro-notes/StructureDefinition-ehealth-careteam-intro.md]: As we are adding ehealth-useContext to CareTeam, I would suggest a slight modification:
"which defines the context(s) in which the CareTeam is intended and expected to be used (non-exclusively)."

File: input/intro-notes/StructureDefinition-ehealth-documentreference-intro.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ In scope of the eHealth infrastructure the primary use of the DocumentReference
 
 See [eHealth-plandefinition](StructureDefinition-ehealth-plandefinition.html) for further details.
 
+The profile is based on the native FHIR Basic resource and adds the following extensions:
```
[the remark this answers is not in the document]
Comment (esn-systematic) [reply input/intro-notes/StructureDefinition-ehealth-documentreference-intro.md]: but here the original description fits (as DocumentReference describes content):
"which defines the context that the content is intended to support"

File: input/intro-notes/StructureDefinition-ehealth-careteam-intro.md (hunk 1 above)
Comment (sys-alex-matic) [reply input/intro-notes/StructureDefinition-ehealth-careteam-intro.md]: > I am not sure if I understand completely. Would this be ok:
> 
> The eHealth profile of CareTeam has the following extensions:
> 
> * `ehealth-useContext` which defines the context that the content is intended to support
> 
> Or would you like more text in the ehealth-useContext description?

Yes, my comment was regarding the wording, suggest we use the same wording as in other places: "The eHealth profile of X has the following extensions:"

**Review by sys-alex-matic — APPROVED** [review]

**Pull request merged**